cmd/ebcli/main.go

```diff
@@ -110,6 +110,9 @@ func txCmd(cdc *amino.Codec) *cobra.Command {
 		Short: "Transactions subcommands",
 	}
 
+	txCmd.PersistentFlags().String("ethereum-chain-id", "", "Ethereum chain ID")
```

Comment:
I'd assign a constant for both `"ethereum-chain-id"` and  `"token-contract-address"` as they are being used multiple times

Reply:
```go 
const (
    FlagEthereumChainID string = "ethereum-chain-id"
    FlagTokenContractAddr string = "token-contract-address"
)
````

Reply:
Added `flags.go` to ethbridge/types https://github.com/cosmos/peggy/pull/86/commits/7e1201a2ec806b75a086bced38f059101798adb5